### PR TITLE
[AuditLogging] add unit tests

### PR DESCRIPTION
**Pull request by tianleh**
*Issue #, if available:*

*Description of changes:*
Add unit tests for audit logging related components.

**Before this change**
- All files
yarn test:jest --coverage

-----------------------------------------------|---------|----------|---------|---------|-------------------
File                                           | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------------------|---------|----------|---------|---------|-------------------
All files                                      |   11.58 |    10.24 |   13.25 |    11.3 |

- Limit the coverage to audit logging directory only.
_zero_

**After this change**
Current coverage.

- All files

yarn test:jest --coverage
-----------------------------------------------|---------|----------|---------|---------|-------------------
File                                           | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------------------|---------|----------|---------|---------|-------------------
All files                                      |   17.39 |    15.35 |   17.19 |   17.23 |

- Limit the coverage to audit logging directory only.

yarn test:jest public/apps/configuration/panels/audit-logging/* --coverage --collectCoverageFrom=public/apps/configuration/panels/audit-logging/*

---------------------------------|---------|----------|---------|---------|------------------------------
File                             | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
---------------------------------|---------|----------|---------|---------|------------------------------
All files                        |   75.16 |    71.43 |   52.63 |   75.16 |
 audit-logging-edit-settings.tsx |   51.16 |    33.33 |   46.15 |   51.16 | 49-50,55-60,69,83-94,106-138
 audit-logging.tsx               |      56 |       75 |   44.44 |      56 | 88-104,132-140,165-204
 code-editor.tsx                 |     100 |      100 |     100 |     100 |
 constants.tsx                   |     100 |      100 |     100 |     100 |
 edit-setting-group.tsx          |   76.47 |    81.82 |   45.45 |   76.47 | 70,83,97-100,106-107,112,119
 types.tsx                       |       0 |        0 |       0 |       0 |
 view-setting-group.tsx          |     100 |       75 |     100 |     100 | 36,40,44
---------------------------------|---------|----------|---------|---------|------------------------------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        handleInvalid={mockHandleInvalid}
+      />
+    );
+
+    expect(component.find(EuiSwitch).prop('checked')).toBeTruthy();
+
+    expect(component.find(EuiComboBox).length).toEqual(2);
+
+    expect(component.find(EuiComboBox).at(0).prop('selectedOptions')).toEqual(
+      audit.disabled_rest_categories?.map(stringToComboBoxOption)
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx:90]: The question mark is not needed here, right?

File: public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    );
+
+    expect(component.find(EuiComboBox).at(1).prop('selectedOptions')).toEqual(
+      compliance.read_ignore_users?.map(stringToComboBoxOption)
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx:94]: The question mark is not needed here, right?

**Review by yuxitang-amzn — APPROVED** [review]

File: public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx (hunk 1 above)
Comment (tianleh) [reply public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx:90]: This is to suppress a warning about "TS2532: Object is possibly 'undefined'."

File: public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx (hunk 2 above)
Comment (tianleh) [reply public/apps/configuration/panels/audit-logging/test/edit-setting-group.test.tsx:94]: same above

**Pull request merged**